test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js

```diff
@@ -95,7 +95,7 @@ describe('ExportMfpToolContent', () => {
 			spyOn(mfpServiceMock, 'getCapabilities').and.returnValue(capabilities);
 			const element = await setup({ ...mfpDefaultState, current: initialCurrent });
 
-			expect(element.shadowRoot.querySelector('#select_layout')).toBeTruthy();
+			expect(element.shadowRoot.querySelector('.layout-button')).toBeTruthy();
```

Comment:
better `expect(element.shadowRoot.querySelectorAll('.layout-button')).toHaveSize(1);`

Let's try to remove all truthy/falsy tests here...